app/Http/Controllers/ContactsController.php

```diff
@@ -511,7 +511,7 @@ public function search(Request $request)
             return;
         }
 
-        $results = SearchHelper::searchContacts($needle, 20, '`contacts`.`created_at`');
+        $results = SearchHelper::searchContacts($needle, 20, '`'.DB::connection()->getTablePrefix().'contacts`.`created_at`');
```

Comment:
@asbiin berk.

Can't we do an helper for this instead?

Reply:
Yes good idea!